Livecheckables/metis.rb

```diff
@@ -0,0 +1,6 @@
+class Metis
+  livecheck do
+    url :homepage
+    regex(/METIS\ stable\ version:\ v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    url "http://glaros.dtc.umn.edu/gkhome/metis/metis/download"
    regex(%r{href=.*?/metis.v?(\d+(?:\.\d+)+)\.t}i)
```

It's probably better to use the download page here, since it contains the link to the stable archive file used in the formula. [I had to include the `/` before the filename here to prevent the regex from matching the `mt-metis` release as well.]

Reply:
Ah okay, thanks!